add-workers/du3-ldc1/overlay/default/kustomization.yaml

```diff
@@ -0,0 +1,22 @@
+---
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+
+resources:
+  - github.com/redhat-partner-solutions/vse-ocp-bases/base/1-worker?ref=main
+
+patchesStrategicMerge:
+  - ./patches/nmstate-config.yaml
+
+patches:
+  - target:
+      kind: BareMetalHost
+      name: bmh-day2-worker1
+    path: ./patches/bmh-du3-ldc1.yaml
+  - target:
+      kind: Provisioning
+      name: provisioning-configuration
+    patch: |-
+      - op: replace
+        path: '/spec/disableVirtualMediaTLS'
```

Comment:
Just a comment:  We should probably monitor in our backlog for this and the particular server vendor model we're talking about here.  I know why we are doing this, but we should ideally report this to the OEM, and then come back to this later to re-enable TLS as a best practice.

Reply:
agree, note that these are the defaults for the node du3-ldc1 (SMC). The bases for for 1 generic worker are set to false by default.